deploy/stack/compose/manifests/template.yaml

```diff
@@ -19,23 +19,23 @@ spec:
             image: quay.io/tinkerbell-actions/image2disk:v1.0.0
             timeout: 600
             environment:
-              DEST_DISK: {{ index .Hardware.Disks 0 }}
+              DEST_DISK: $DISK_DEVICE
```

Comment:
im curious, why the move to an env var over using the value in the hardware manifest?

Reply:
this is great feedback, thank you! im good with this.